content/blogposts/2023/cody-vscode-0-18-release.md

```diff
@@ -0,0 +1,154 @@
+---
+title: "Cody for VS Code v0.18 release"
+authors:
+  - name: Taylor Sperry
+    url: https://handbook.sourcegraph.com/team/#taylor-sperry
+  - name: Tim Lucas
+    url: https://handbook.sourcegraph.com/team/#tim-lucas
+  - name: Kalan Chan
+    url: https://handbook.sourcegraph.com/team/#kalan-chan
+  - name: Justin Dorfman
+    url: https://handbook.sourcegraph.com/team/#justin-dorfman
+  
+publishDate: 2023-12-05T10:00-01:00
+description: "The Cody for VS Code v0.18.0 update introduces a new chat interface, allowing for side-by-side chat windows and improved context control, and more."
+tags: [blog]
+slug: "cody-vscode-0-18-release"
+published: true
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/blog-vscode-v018-release/cody-vscode-v0-18-og.png
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/blog-vscode-v018-release/cody-vscode-v0-18-og.png
+---
+
+The Cody for VS Code v0.18.0 update introduces a new chat interface, allowing for side-by-side chat windows and improved context control, including a reworked context dropdown that shows file names and line ranges, and an Enhanced Context button for better LLM responses.
+
+
+## New Chat Interface
+
+Chats now open alongside your code, in a larger window, without needing the Cody sidebar open. This means you can have multiple chats open side-by-side and configure them to give more space to read longer code blocks. You can also manage your chats directly in the sidebar.
+
+<Figure 
+  src="https://storage.googleapis.com/sourcegraph-assets/blog/blog-vscode-v018-release/blog-v018-new-chat-interface-001.gif"
+  alt="Cody for VS Code v0.18.0 Release"
+  caption="Optional image caption"
+/>
+
+
+
+## Context Controls
+
+Cody’s chat view has a completely reworked context dropdown that shows not just the file names but also the exact line ranges that Cody used to provide a chat response. This helps you understand exactly what additional context was included. You can also click any of the context items to open the file with line range selected.
+
+<Figure 
+  src="https://storage.googleapis.com/sourcegraph-assets/blog/blog-vscode-v018-release/blog-v018-context-controls-002.gif"
+  alt="Cody for VS Code v0.18.0 Release"
+  caption="Optional image caption"
+/>
+
+
+You can now control whether Cody fetches additional context (e.g. Embeddings), using the new Enhanced Context button in the chat message text input. When enabled, Cody fetches embeddings and includes them with your message to get the best responses from the LLM. When disabled, only @-included files or symbols with be included as context.
+
+
+<Figure 
+  src="https://storage.googleapis.com/sourcegraph-assets/blog/blog-vscode-v018-release/blog-v018-mention-003.png"
```

Comment:
I think this image was supposed to be after line 59, for the @ command stuff. 

All the images after that are also in the wrong order!